tests/functional/incremental-fetching-datetime/expected/data/out/tables/in.c-main.auto-increment-timestamp.csv.manifest

```diff
@@ -0,0 +1,287 @@
+            },
+            {
+                "key": "KBC.uniqueKey",
+                "value": false
+            },
+            {
+                "key": "KBC.ordinalPosition",
+                "value": 3
+            }
+        ],
+        "somedecimal": [
+            {
+                "key": "KBC.datatype.type",
+                "value": "numeric"
+            },
+            {
+                "key": "KBC.datatype.nullable",
+                "value": true
+            },
+            {
+                "key": "KBC.datatype.basetype",
+                "value": "NUMERIC"
+            },
+            {
+                "key": "KBC.datatype.length",
+                "value": "10,8"
+            },
+            {
+                "key": "KBC.datatype.default",
+                "value": "10.2"
+            },
+            {
+                "key": "KBC.sourceName",
+                "value": "somedecimal"
+            },
+            {
+                "key": "KBC.sanitizedName",
+                "value": "somedecimal"
+            },
+            {
+                "key": "KBC.primaryKey",
+                "value": false
+            },
+            {
+                "key": "KBC.uniqueKey",
+                "value": false
+            },
+            {
+                "key": "KBC.ordinalPosition",
+                "value": 4
+            }
+        ],
+        "type": [
+            {
+                "key": "KBC.datatype.type",
+                "value": "character varying"
+            },
+            {
+                "key": "KBC.datatype.nullable",
+                "value": true
+            },
+            {
+                "key": "KBC.datatype.basetype",
+                "value": "STRING"
+            },
+            {
+                "key": "KBC.datatype.length",
+                "value": "55"
+            },
+            {
+                "key": "KBC.datatype.default",
+                "value": ""
+            },
+            {
+                "key": "KBC.sourceName",
+                "value": "type"
+            },
+            {
+                "key": "KBC.sanitizedName",
+                "value": "type"
+            },
+            {
+                "key": "KBC.primaryKey",
+                "value": false
+            },
+            {
+                "key": "KBC.uniqueKey",
+                "value": false
+            },
+            {
+                "key": "KBC.ordinalPosition",
+                "value": 5
+            }
+        ],
+        "datetime": [
+            {
+                "key": "KBC.datatype.type",
+                "value": "timestamp without time zone"
+            },
+            {
+                "key": "KBC.datatype.nullable",
+                "value": false
+            },
+            {
+                "key": "KBC.datatype.basetype",
+                "value": "TIMESTAMP"
+            },
+            {
+                "key": "KBC.datatype.default",
+                "value": "(now"
```

Comment:
toto default value vyzera nejak divne.

Reply:
jj ale je to tak - to už tam bude dlouho, takže bych to do tohodle PR nedával:
![image](https://user-images.githubusercontent.com/12143866/106771020-d73c7080-663e-11eb-8339-c2d9c9e7bc87.png)

a když se na to udělá:
https://github.com/keboola/db-extractor-redshift/blob/master/src/Extractor/RedshiftMetadataProvider.php#L174

tak s toho zůstane to co tu je
stačilo by z toho odstranit ještě tu závorku, ale to bych nechal na samostatný PR

Reply:
ok